### PR TITLE
Add toString() method to BlockEncodingBuffers and ColumnarXXX

### PR DESCRIPTION
**Pull request by yingsu00**
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-spi/src/main/java/com/facebook/presto/spi/block/ColumnarArray.java [hunk 1]
```diff
@@ -155,4 +155,15 @@ public long getRetainedSizeInBytes()
     {
         return nullCheckBlock.getRetainedSizeInBytes() + elementsBlock.getRetainedSizeInBytes() + sizeOf(offsets);
     }
+
+    public String toString()
```
Comment (mbasmanova) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/ColumnarArray.java]: @Override

File: presto-spi/src/main/java/com/facebook/presto/spi/block/ColumnarMap.java [hunk 2]
```diff
@@ -191,4 +191,17 @@ public long getRetainedSizeInBytes()
     {
         return nullCheckBlock.getRetainedSizeInBytes() + keysBlock.getRetainedSizeInBytes() + valuesBlock.getRetainedSizeInBytes() + sizeOf(offsets) + sizeOf(hashTables);
     }
+
+    public String toString()
```
Comment (mbasmanova) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/ColumnarMap.java]: @Override

File: presto-spi/src/main/java/com/facebook/presto/spi/block/ColumnarRow.java [hunk 3]
```diff
@@ -153,4 +153,16 @@ public long getRetainedSizeInBytes()
         }
         return nullCheckBlock.getRetainedSizeInBytes() + fieldsRetainedSize;
     }
+
+    public String toString()
```
Comment (mbasmanova) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/ColumnarRow.java]: @Override

File: presto-spi/src/main/java/com/facebook/presto/spi/block/ColumnarRow.java [hunk 4]
```diff
@@ -153,4 +153,16 @@ public long getRetainedSizeInBytes()
...
+    {
+        StringBuilder sb = new StringBuilder("ColumnarRow{");
+        sb.append("positionCount=").append(getPositionCount()).append(",");
+        sb.append("fields count=").append(fields.length).append(",");
```
Comment (mbasmanova) [inline presto-spi/src/main/java/com/facebook/presto/spi/block/ColumnarRow.java]: fieldsCount for consistency

File: presto-main/src/main/java/com/facebook/presto/operator/repartition/BlockEncodingBuffer.java [hunk 5]
```diff
@@ -51,4 +51,6 @@
     long getRetainedSizeInBytes();
 
     long getSerializedSizeInBytes();
+
+    String toString();
```
Comment (mbasmanova) [inline presto-main/src/main/java/com/facebook/presto/operator/repartition/BlockEncodingBuffer.java]: not needed

File: presto-main/src/main/java/com/facebook/presto/operator/repartition/DecodedBlockNode.java [hunk 6]
```diff
@@ -79,4 +79,17 @@ else if (decodedBlock instanceof ColumnarRow) {
         }
         return size;
     }
+
+    public String toString()
```
Comment (mbasmanova) [inline presto-main/src/main/java/com/facebook/presto/operator/repartition/DecodedBlockNode.java]: @Override

File: presto-main/src/main/java/com/facebook/presto/operator/repartition/Int128ArrayBlockEncodingBuffer.java [hunk 7]
```diff
@@ -109,6 +109,15 @@ public long getSerializedSizeInBytes()
                 getNullsBufferSerializedSizeInBytes();    // nulls buffer
     }
 
+    @Override
+    public String toString()
+    {
+        StringBuilder sb = new StringBuilder("Int128ArrayBlockEncodingBuffer{");
```
Comment (mbasmanova) [inline presto-main/src/main/java/com/facebook/presto/operator/repartition/Int128ArrayBlockEncodingBuffer.java]: better Int128ArrayBlockEncodingBuffer.class.getSimpleName(); ditto other places

**Comment by yingsu00** [comment]
> @yingsu00 Looks fine. What's the motivation for this change? Is this useful for debugging? Since only RLE and Dictionary blocks implement toString(), what does the output look like for other blocks?

It's useful for debugging. Actually all blocks implement toString(), but RLE one was missing a "{" and Dictionary blocks didn't show the dictionary. Commit `Improve toString() for DictionaryBlock and RunLengthEncodedBlock` is to improve the implementation. 

Similarly, adding toString() to BlockEncodingBuffers and ColumnarXXX is also very useful for debugging

File: presto-main/src/main/java/com/facebook/presto/operator/repartition/Int128ArrayBlockEncodingBuffer.java (hunk 7 above)
Comment (yingsu00) [reply presto-main/src/main/java/com/facebook/presto/operator/repartition/Int128ArrayBlockEncodingBuffer.java]: Sure. Changed to
```
StringBuilder sb = new StringBuilder(getClass().getSimpleName()).append("{");
```

**Comment by yingsu00** [comment]
@mbasmanova Thanks for reviewing. I have updated the PR with all comments addressed.

**Pull request merged**